.readthedocs.yaml

```diff
@@ -8,9 +8,9 @@ python:
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
+    python: "3"
   jobs:
-    pre_build:
-      - echo "Placeholder for doxygen builds via cmake"
+    post_create_environment:
+      - pip install --pre --upgrade sphinx-book-theme
```

Comment:
If I understand correctly, this line installs `sphinx-book-theme==v1.1.0rc1` (sphinx-7 is supported), right?

Reply:
Yes, [here](https://readthedocs.org/projects/spglib/builds/22791514/) is the build. Basically we do not have a pin for the `sphinx-book-theme`, so when RTD detects that `sphinx-book-theme` was already installed (right after the environment was created), it uses that one. The `--pre` ensures we are pricking up `rc` releases (Also a reason why I would prefer on the spglib side to always upload to `PyPI` instead of https://test.pypi.org/legacy/)